skeleton/common/tsconfig.json__if_typescript

```diff
@@ -44,6 +42,8 @@
     "lib": ["es2015", "dom"],
     "moduleResolution": "node",
     "baseUrl": "src",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
I like `"esModuleInterop": true,` personally, but majority of Aurelia core team do not like this switch. I don't think you can pass review :-)

Reply:
Bad switch. :)

Reply:
@Sayan751 fight or run? :-)

Reply:
I prefer this as it let me import json as a module, and I can work with this object as a normal JS object. For readonly config, or translation files (i18n) it is quite handy. I am using it for quite a some time now. Till now, have not faced any issue. Would be great if you have some use cases where this can be problematic. This info will be very important to me personally to evaluate the usages in my project 😄

Reply:
errr... `import * as json from 'a.json';` does not work with `"esModuleInterop": false`?

Reply:
I started with `resolveJsonModule` only, but it didn't work alone. Thereafter I use both of them together (muscle memory 😛). However, this info is also quite a few months old. Not sure, if it works without `esModuleInterop`.

Reply:
I found the issue to be closed: https://github.com/Microsoft/TypeScript/issues/25400

I will try without `esModuleInterop`.

Reply:
Removed the `esModuleInterop`. The JSON import is working corrctly.